packages/strapi-design-system/src/Combobox/Combobox.tsx

```diff
@@ -30,9 +30,12 @@ export interface ComboboxInputProps
       | 'defaultTextValue'
       | 'required'
       | 'disabled'
+      | 'isPrintableCharacter'
     >,
-    Pick<FieldProps, 'error' | 'id'> {
+    Pick<FieldProps, 'error' | 'id'>,
+    Omit<ComboboxPrimitive.TextInputProps, 'required' | 'disabled' | 'value' | 'onChange' | 'size'> {
   children: React.ReactNode;
+  className?: string;
```

Comment:
Do we really need that? Imo that is a very uncommon pattern in our codebase and I'm wondering if the Combobox should somehow translate between Radix and the components using it instead of exposing such a low level prop.

Reply:
> if the Combobox should somehow translate between Radix and the components using it

I'm not sure what you're suggesting here.

To answer your question, we have it because the DTP has a 70/30 flex split with the DatePicker & TimePicker (respectively), I suppose we could style the `Flex` to declare it but I find that to be more brittle than using an API which is endorsed by the library https://styled-components.com/docs/basics#extending-styles 

In response to it being uncommon in the codebase – that's why we have over-engineered dom code because we're for some reason opposed to this API – I don't think there's one answer over the other, i just think it's best to use them interchangeably based on the use-case 😅